tests/integration/cli/action/test_action_classification.py

```diff
@@ -13,13 +13,11 @@
 from tests.test_suite.e2e_test_system import e2e_pytest_component
 
 # Finetuning arguments
+# TODO: Need to change sample dataset
 args = {
-    "--train-ann-file": "data/custom_action_recognition/custom_dataset/train_list_rawframes.txt",
-    "--train-data-roots": "data/custom_action_recognition/custom_dataset/rawframes",
-    "--val-ann-file": "data/custom_action_recognition/custom_dataset/train_list_rawframes.txt",
-    "--val-data-roots": "data/custom_action_recognition/custom_dataset/rawframes",
-    "--test-ann-files": "data/custom_action_recognition/custom_dataset/train_list_rawframes.txt",
-    "--test-data-roots": "data/custom_action_recognition/custom_dataset/rawframes",
+    "--train-data-roots": "data/datumaro/cvat_dataset/action_classification/train",
+    "--val-data-roots": "data/datumaro/cvat_dataset/action_classification/train",
+    "--test-data-roots": "data/datumaro/cvat_dataset/action_classification/train",
```

Comment:
is it intentional to define `"data/datumaro/cvat_dataset/action_classification/train",` for `val` and `test` as the root directory?

Reply:
Currently, we have no data for val and test. And the purpose of CLI test isn't to check the performance metric. So, I think using the same data won't be a problem.

Actually, we'll replace the current dataset to public dataset (Kinetics, AVA) in the near future. So, this data is just temporary.